ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -220,8 +220,8 @@ Sometimes, you just can't get ActiveRecord to do what you want it to. In that ca
 
 1. Read the first 6 sections of the [Rails Guide on Active Record Querying](http://guides.rubyonrails.org/active_record_querying.html) for a more basic overview of query functions. Don't worry too much about batching and `#find_each`.
 2. Read section 20 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects) for a brief look at using `exists?`, `any?` and `many?`.
-3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
-4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).
+3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#grouping) for an understanding of aggregate functions and the calculations you can run on them.
+4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions). 
```

Comment:
Why are these changes here?

Reply:
By adding #grouping, the link'll point to the specific section where we want the student starts reading instead of letting him looking for the section from the top of the page.

Reply:
Ah I see.

This looks like you have some changes that are leaking over from #26955 . If that's the case, you should change this file so it reflects what's currently in `main` and leave the changes around these links to that PR instead of this one.

This is just to avoid introducing conflicts between your different PRs (which tend to be a bit annoying to fix).